src/api/orderProducts/orderProducts.repository.js

```diff
@@ -0,0 +1,18 @@
+import orderProductsModel from './orderProducts.model.js';
+
+async function getProductGameTitleFromOrder({ orderId }) {
+  const productGameTitleFromOrder = await orderProductsModel
+    .find({ orderId })
+    .populate({
+      path: 'productId',
+    })
+    .limit(1)
+    .lean();
+
+  return productGameTitleFromOrder[0];
```

Comment:
You have 'limit' set to 1, but also take the first item from an array

Reply:
I think it's correct.
- The "limit" causes the return to he an array with one element, so we dont't get unnecessarily large data from the database. But it's still an array, so...
- We return the [0] element instead of returning it wrapped in an array.

Reply:
Ahuanmarca got the right answer :)

Reply:
Ah, I see. But, in that case, why don't you use the method .findOne?

Reply:
resolved using findOne :)